views/index.html

```diff
@@ -0,0 +1,274 @@
+            <div>Page view count:<span id="count-value">
+            <%= pageCountMessage %>
+            </span></div>
+          </hgroup>
+
+
+        <div class="row">
+          <section class='col-xs-12 col-sm-6 col-md-6'>
+            <section>
+              <h2>Deploying code changes</h2>
+                <p>OpenShift uses the <a href="http://git-scm.com/">Git version control system</a> for your source code, and grants you access to it via the Secure Shell (SSH) protocol. In order to upload and download code to your application you need to give us your <a href="https://developers.openshift.com/en/managing-remote-connection.html">public SSH key</a>. You can upload it within the web console or install the <a href="https://developers.openshift.com/en/managing-client-tools.html">RHC command line tool</a> and run <code>rhc setup</code> to generate and upload your key automatically.</p>
+
+                <h3>Working in your local Git repository</h3>
+                <p>If you created your application from the command line and uploaded your SSH key, rhc will automatically download a copy of that source code repository (Git calls this 'cloning') to your local system.</p>
+
+                <p>If you created the application from the web console, you'll need to manually clone the repository to your local system. Copy the application's source code Git URL and then run:</p>
+
+<pre>$ git clone &lt;git_url&gt; &lt;directory_to_create&gt;
+
+# Within your project directory
+# Commit your changes and push to OpenShift
+
+$ git commit -a -m 'Some commit message'
+$ git push</pre>
+
+
+                  <ul>
+                    <li><a href="https://developers.openshift.com/en/managing-modifying-applications.html">Learn more about deploying and building your application</a></li>
+                    <li>See the README file in your local application Git repository for more information on the options for deploying applications.</li>
+                  </ul>
+            </section>
+
+          </section>
+          <section class="col-xs-12 col-sm-6 col-md-6">
+
+                <h2>Managing your application</h2>
+
+                <h3>Web Console</h3>
+                <p>You can use the OpenShift web console to enable additional capabilities via cartridges, add collaborator access authorizations, designate custom domain aliases, and manage domain memberships.</p>
+
+                <h3>Command Line Tools</h3>
+                <p>Installing the <a href="https://developers.openshift.com/en/managing-client-tools.html">OpenShift RHC client tools</a> allows you complete control of your cloud environment. Read more on how to manage your application from the command line in our <a href="https://www.openshift.com/user-guide">User Guide</a>.
+                </p>
+
+<h2>Development Resources</h2>
+                  <ul>
+                    <li><a href="https://developers.openshift.com/en/node-js-overview.html">Getting Started with Node.js on OpenShift</a></li>
```

Comment:
Most of these link are v2 oriented. I CakePHP I've got rid of them and replaced them with link to:
- OpenShiftV3 Documentation
- Openshift Origin GitHub
- STI GitHub
- Stack Overflow questions for OpenShift
- Git documentation

https://github.com/openshift/cakephp-ex/blob/master/app/View/Layouts/default.ctp#L287-L291